### ISSUE TITLE
Build Issue

### ISSUE THREAD

**Issue by nreddipalle** [upsidetravel/bucket-antivirus-function#36]
I am unable to get the build work. Getting the below error when running MAKE. any help??

![image](https://user-images.githubusercontent.com/28302399/42000526-ed029ea2-7a2e-11e8-9b3d-4e0d87ef0fb1.png)

**Comment by kevitan** [comment]
I'm also getting an error on build. Running MacOS High Sierra 10.13.4.

after pulling the docker images, i get to here:

```
Install  1 Package

Total download size: 263 k
Installed size: 800 k
Downloading packages:
zip-3.0-11.amzn2.x86_64.rpm                                                                                                                                                                                            | 263 kB  00:00:00     
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Installing : zip-3.0-11.amzn2.x86_64                                                                                                                                                                                                    1/1 
  Verifying  : zip-3.0-11.amzn2.x86_64                                                                                                                                                                                                    1/1 

Installed:
  zip.x86_64 0:3.0-11.amzn2                                                                                                                                                                                                                   

Complete!
./build_lambda.sh: line 23: pip: command not found
make: *** [archive] Error 127
```

but I check:
```
 which pip
/Users/itsme/.pyenv/shims/pip
```

**Comment by Lacan82** [comment]
@nreddipalle this is because the repo doesn't have clamav in it...
@kevitan yes you have it locally but when make runs it launched the docker image, and needs to install there, edit build_lambda.sh and change python27-pip to python2-pip.

**Comment by denniswebb** [comment]
Fixed by #57
